### PR TITLE
Feature/v6 create stored card from raw data

### PR DESCRIPTION
**Pull request by sponglord**
## Summary
To align with the other SDKS - allow the possibility of creating a standalone storedCard with an object pulled directly from the `storedPaymentMethods` array from the `/paymentMethods` response (instead of having to use a "processed" object from `checkout.paymentMethodsResponse.storedPaymentMethods`

## Tested scenarios
All existing unit tests run
New unit tests added

### CODE REVIEW

**Review by ribeiroguilherme — APPROVED** [review]
I have mixed feelings about this feature honestly.  If we do for Cards, shouldn't we do for all other stored payment methods as well? I am not much in favor of supporting only the Card component in this case, although the scope of the changes would be much bigger.. 

Is this something requested by merchants, or just an idea that came up in our internal discussions?

File: packages/e2e/app/src/pages/StoredCards/StoredCards.js [hunk 1]
```diff
@@ -34,8 +34,8 @@ const initCheckout = async () => {
         ...window.cardConfig
     };
 
-    // Credit card with installments
-    window.storedCard = new Card(checkout, { ...storedCardData }).mount('.stored-card-field');
+    // Stored Credit card
+    window.storedCard = new Card({ core: checkout, ...storedCardData }).mount('.stored-card-field');
```
[the remark this answers is not in the document]
Comment (sponglord) [reply packages/e2e/app/src/pages/StoredCards/StoredCards.js]: Fixed

**Comment by sponglord** [comment]
> I have mixed feelings about this feature honestly. If we do for Cards, shouldn't we do for all other stored payment methods as well? I am not much in favor of supporting only the Card component in this case, although the scope of the changes would be much bigger..
> 
> Is this something requested by merchants, or just an idea that came up in our internal discussions?

It came up in internal discussions. This aligns us with how the other SDKs allow merchants to create a storedCard.
tbh - I think it makes  more sense to be able to do it with the raw `/paymentMethods` response data; rather than waiting for checkout to "process" it (which basically just adds the `storedPaymentMethodId`)

However I do agree that it would make sense to do it for other stored PMS

**Comment by ribeiroguilherme** [comment]
Yea it does make sense to create directly without pre-processing.. Maybe we don't even part of the pre-processing logic anymore, if we start adding this by default to the components

**Pull request merged**